### PR TITLE
feat: add simple_math directory with arithmetic operations

### LINKED ISSUE #15: Add simple_math directory with a Cairo script with arithmetic operations
## Description
- We want to have a similar directory like the `hello_world` but called `simple_math`
- This directory should hold a cairo file that performs simple math operations and print results
- Follow this [guide](https://book.cairo-lang.org/ch01-02-hello-world.html) to replicate the work done in the `hello_world` directory.

### PR DESCRIPTION
**Pull request by derianrddev**
# Pull Request

## Issue
- Closes #15 

## Description
1. Creation of the `simple_math` directory:  
   The `simple_math` directory was created at the path `cairo/scripts/simple_math`.

2. File `simple_math.cairo`:
   A new file named `simple_math.cairo` was added to the `src` directory of `simple_math`.

3. Arithmetic Functions:  
   Functions for each basic arithmetic operation were implemented:
   - Addition (`+`)
   - Subtraction (`-`)
   - Multiplication (`*`)
   - Division (`/`)
   - Modulus (`%`)

4. Function `main`:  
   A `main` function was created with examples to execute all operations and display the results.

## Screenshots
image

## Additional Notes
In the division example, I included a clarification that says “in Cairo, the division returns only the integer part, no decimals”. Since these examples are designed for newcomers to the Cairo ecosystem, I thought it was a good idea to provide this additional information to facilitate their understanding.

### CODE REVIEW

File: cairo/scripts/simple_math/src/simple_math.cairo [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    // Multiply example
+    println!("{} * {} = {}", number1, number2, multiply(number1, number2));
+
+    // Divide example
+    println!("{} / {} = {} (in Cairo, division returns only the integer part, without decimals)", number1, number2, divide(number1, number2));
+
+    // Modulus example
+    println!("{} % {} = {}", number1, number2, modulus(number1, number2));
+}
```
Comment (danielcdz) [inline cairo/scripts/simple_math/src/simple_math.cairo:41]: Can you add a final line by the EOF?

File: cairo/scripts/simple_math/src/simple_math.cairo [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+fn modulus(number1: u128, number2: u128) -> u128 {
+    assert(number2 != 0, 'Modulus by zero is not allowed');
+    number1 % number2
+}
+
+fn main() {
+    let number1: u128 = 22;
+    let number2: u128 = 7;
+
+    // Sum example
```
Comment (danielcdz) [inline cairo/scripts/simple_math/src/simple_math.cairo:27]: Can you print those comments before every operation? For example In this case print `-> Sum Example:` and the same with the others?

**Comment by danielcdz** [comment]
@derianrddev Good PR overall excellent work! 🫡 
Just left some minor comments

**Comment by derianrddev** [comment]
@danielcdz  Ready, I have made the changes

**Review by danielcdz — APPROVED** [review]
LGTM!

**Pull request merged**